internal/controller/pkg/revision/hook.go

```diff
@@ -161,13 +170,6 @@ func NewConfigurationHooks() *ConfigurationHooks {
 
 // Pre sets status fields based on the configuration package.
 func (h *ConfigurationHooks) Pre(ctx context.Context, pkg runtime.Object, pr v1.PackageRevision) error {
-	_, ok := pkg.(*pkgmeta.Configuration)
-	if !ok {
-		return errors.New(errNotConfiguration)
-	}
-
-	// TODO(hasheddan): update any status fields relevant to package revisions
-
```

Comment:
I noticed we were checking this type assertion in the pre hook but not the post hook, where both are no-ops. I figured it was safe to remove this rather than updating it given that we already perform this check in the linter.